wgpu-types/src/lib.rs

```diff
@@ -881,56 +1187,90 @@ pub enum SwapChainStatus {
     OutOfMemory,
 }
 
+/// Operation to perform to the output attachment at the start of a renderpass.
+///
+/// This must be clear if it is the first renderpass rendering to a swap chain image.
 #[repr(C)]
 #[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
 #[cfg_attr(feature = "trace", derive(Serialize))]
 #[cfg_attr(feature = "replay", derive(Deserialize))]
 #[cfg_attr(feature = "peek-poke", derive(PeekPoke))]
 pub enum LoadOp {
```

Comment:
No option for "I don't care what was in this attachment before the renderpass began"?

Reply:
That option is `Load`, and either way this is a problem for upstream not here.